atlassian_jwt_auth/frameworks/django/backend.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings as django_settings
+from django.http import HttpResponse, HttpResponseForbidden
+from ..common.backend import Backend
+from ..common.utils import SettingsDict
+
+
+class DjangoBackend(Backend):
+    def get_authorization_header(self, request=None):
+        if request is None:
+            raise ValueError('No request available')
+
+        return request.META.get('HTTP_AUTHORIZATION', b'')
+
+    def get_401_response(self, data=None, headers=None):
+        if headers is None:
+            headers = {}
+
+        headers.update(self.default_headers_401)
+
+        response = HttpResponse(content=data, status=401)
+        for k, v in headers.items():
+            response[k] = v
+
+        return response
+
+    def get_403_response(self, data=None, headers=None):
+        if headers is None:
+            headers = {}
+
+        response = HttpResponseForbidden(data)
+        for k, v in headers.items():
+            response[k] = v
+
+        return response
+
+    def set_asap_claims_for_request(self, request, claims):
+        request.asap_claims = claims
+
+    @property
```

Comment:
So settings should not change?

Reply:
Outside of test situations, the settings provided by a backend should be the same no matter how many times you call this.